DragaliaAPI/Services/Game/SessionService.cs

```diff
@@ -139,15 +144,20 @@ public async Task<string> ActivateSession(string idToken)
         return session.SessionId;
     }
 
-    public async Task<string> CreateSession(string idToken, string accountId, long viewerId)
+    public async Task<string> CreateSession(
+        string idToken,
+        string accountId,
+        long viewerId,
+        DateTimeOffset loginTime
+    )
     {
         // Check for existing session
         Session? existingSession = await this.TryLoadSession(Schema.Session_IdToken(idToken));
         if (existingSession is not null)
             return existingSession.SessionId;
```

Comment:
If there's an existing session it may not refresh the login time? 🤔 

Maybe best just to ignore any existing sessions and always issue a new one. That's what the official game did iirc